draft-ietf-oauth-identity-chaining.md

```diff
@@ -133,6 +135,23 @@ The flow illustrated in Figure 1 shows the steps the client in trust Domain A ne
 ## Authorization Server Discovery
 This specification does not define authorization server discovery. A client MAY maintain a static mapping or use other means to identify the authorization server. The `authorization_servers` property in {{I-D.ietf-oauth-resource-metadata}} MAY be used.
 
+## Sender Constrained Tokens
+Either authorization server MAY issue sender constrained tokens. There are currently two options for sender constrained tokens: Mutually Authenticated TLS (mTLS) {{RFC8705}} and Demonstrating Proof of Possessions (DPoP) {{RFC9449}}.
+
+This document introduces the following authorization server metadata values to indicate the authorization server's support for DPoP or mTLS sender constrained tokens.
+
+requested_cnf_supported_dpop
+: OPTIONAL. Boolean indicator of whether the authorization server supports DPoP sender constrained tokens.
+When the authorization server is acting as the Client, the Client MAY check whether AS-B supports sender constrained tokens by obtaining the authorization server metadata and verifying "requested_conf_supported" is set to true.
+
+requested_cnf_supported_mtls 
+: OPTIONAL. Boolean indicator of whether the authorization server supports mTLS sender constrained tokens.
+When the authorization server is acting as the Client, the Client MAY check whether AS-B supports sender constrained tokens by obtaining the authorization server metadata and verifying the appropriate sender constrained token method is set to true.
+
+When the authorization server is acting as the Client, the Client MAY check whether AS-B supports sender constrained tokens by obtaining the authorization server metadata and verifying the appropriate sender constrained token method is set to true.
+
+If the inbound token is sender constrained, RS-A MUST verify the proof of possession for the inbound token before proceeding to the Token Exchange Request.
```

Comment:
I'm not sure about this whole section TBH for a number of reasons that escape my ability to articulate easily or meaningfully.